docs/cudf/source/cudf_polars/index.rst

```diff
@@ -0,0 +1,38 @@
+cuDF-based GPU backend for Polars [Open Beta]
+=============================================
+
+cuDF supports an in-memory, GPU-accelerated execution engine for Python users of the Polars Lazy API. 
+The engine supports most of the core expressions and data types as well as a growing set of more advanced dataframe manipulations 
+and data file formats. When using the GPU engine, Polars will convert expressions into an optimized query plan and determine 
+whether the plan is supported on the GPU. If it is not, the execution will transparently fall back to the standard Polars engine 
+and run on the CPU. 
+
+Benchmark
+---------
+We reproduced the `Polars Decision Support (PDS) <https://github.com/pola-rs/polars-benchmark>`_ benchmark to compare Polars GPU engine with the default CPU settings across several dataset sizes. Here are the results:
```

Comment:
Use two underscores. We want to use anonymous references in this document. See https://docs.anaconda.com/restructuredtext/#external-links

> A single underscore _ creates a hyperlink reference target for the external link, while a double underscore __ creates an anonymous reference. It is recommended to use a single underscore in most instances. However, if you have more than one instance of the same link text in your document, this will cause a duplicate reference error. Using a double underscore instead to anonymize the reference is a way to remove this error (if you can’t change the wording of the link itself).

(I disagree with the "recommended to use a single underscore in most instances" piece, it always causes more trouble than it's worth. I always use two underscores.)

```suggestion
We reproduced the `Polars Decision Support (PDS) <https://github.com/pola-rs/polars-benchmark>`__ benchmark to compare Polars GPU engine with the default CPU settings across several dataset sizes. Here are the results:
```